modules/coreos-layering-configuring.adoc

```diff
@@ -97,20 +97,20 @@ $ oc get mc
 [source,terminal]
 ----
 NAME                                               GENERATEDBYCONTROLLER                      IGNITIONVERSION   AGE
-00-master                                          5bdb57489b720096ef912f738b46330a8f577803   3.2.0             95m
-00-worker                                          5bdb57489b720096ef912f738b46330a8f577803   3.2.0             95m
-01-master-container-runtime                        5bdb57489b720096ef912f738b46330a8f577803   3.2.0             95m
-01-master-kubelet                                  5bdb57489b720096ef912f738b46330a8f577803   3.2.0             95m
-01-worker-container-runtime                        5bdb57489b720096ef912f738b46330a8f577803   3.2.0             95m
-01-worker-kubelet                                  5bdb57489b720096ef912f738b46330a8f577803   3.2.0             95m
-99-master-generated-registries                     5bdb57489b720096ef912f738b46330a8f577803   3.2.0             95m
+00-master                                          5bdb57489b720096ef912f738b46330a8f577803   3.4.0             95m
+00-worker                                          5bdb57489b720096ef912f738b46330a8f577803   3.4.0             95m
+01-master-container-runtime                        5bdb57489b720096ef912f738b46330a8f577803   3.4.0             95m
+01-master-kubelet                                  5bdb57489b720096ef912f738b46330a8f577803   3.4.0             95m
+01-worker-container-runtime                        5bdb57489b720096ef912f738b46330a8f577803   3.4.0             95m
+01-worker-kubelet                                  5bdb57489b720096ef912f738b46330a8f577803   3.4.0             95m
+99-master-generated-registries                     5bdb57489b720096ef912f738b46330a8f577803   3.4.0             95m
 99-master-ssh                                                                                 3.2.0             98m
-99-worker-generated-registries                     5bdb57489b720096ef912f738b46330a8f577803   3.2.0             95m
+99-worker-generated-registries                     5bdb57489b720096ef912f738b46330a8f577803   3.4.0             95m
 99-worker-ssh                                                                                 3.2.0             98m
```

Comment:
*PEER REVIEW* I verified by testing that these two machine configs remain at 3.2.0.